web/src/lib/services/api.ts

```diff
@@ -153,6 +153,19 @@ export function getProfile(tokenValue, successCB: (User) => void) {
         });
 }
 
+export function getStore(storeName: string, successCB: (User) => void, errorHandler = new ErrorHandler()) {
```

Comment:
This whole function is essentially the same with `getProfile`, except the URL differs.

Reply:
They are quite different -
getProfile gets only yourself and only if you have a token and its always the same endpoint
getStall gets any user , whether or not you are logged in and takes an argument from the url to do this